src/cfnlint/decode/__init__.py

```diff
@@ -91,6 +94,12 @@ def create_match_yaml_parser_error(parser_error, filename):
         cfnlint.ParseError(), message=msg)
 
 
+def create_match_file_error(filename, msg):
+    """Create a Match for a parser error"""
+    return cfnlint.Match(
+        1, 1, 1, 2, filename, cfnlint.ParseError(), message=msg)
```

Comment:
this function call is a bit cryptic. Should it used named parameters?

Reply:
Updated.  Thanks!